blocklist-client/src/client/sanctions.rs

```diff
@@ -0,0 +1,309 @@
+//! This module interacts with the risk API to determine the risk severity associated with a user wallet address.
+//!
+//! It provides functionality to:
+//! - Check if given address is under sanctions
+//!
+//! The module includes functions to handle API requests, interpret responses, and map them to application-specific errors.
+
+use crate::common::error::Error;
+use crate::common::{BlocklistStatus, RiskAssessment, RiskSeverity};
+use crate::config::RiskAnalysisConfig;
+use reqwest::{Client, Response, StatusCode};
+use serde::Deserialize;
+use std::error::Error as StdError;
+use tracing::debug;
+const API_BASE_PATH: &str = "/api/v1/address";
+
+/// Represents the identification information for a blockchain address.
+#[derive(Debug, Deserialize)]
+pub struct Identification {
+    /// The Chainalysis Entity category. For sanctioned addresses, the value will be 'sanctions'.
+    #[serde(rename = "category")]
+    pub _category: String,
+    /// The OFAC name associated with the sanctioned address.
+    #[serde(rename = "name")]
+    pub _name: Option<String>,
+    /// The OFAC description of the sanctioned address.
+    #[serde(rename = "description")]
+    pub _description: Option<String>,
+    /// The OFAC URL for more information about the sanctioned address.
+    #[serde(rename = "url")]
+    pub _url: Option<String>,
+}
+
+/// Response structure of the sanctions API
+#[derive(Debug, Deserialize)]
+pub struct SanctionsResponse {
+    /// Array with sanctions data. If empty, the address is not blocklisted.
+    pub identifications: Vec<Identification>,
+}
+
+fn risk_assessment_path(base_url: &str, address: &str) -> String {
+    format!("{}{}/{}", base_url, API_BASE_PATH, address)
```

Comment:
Nit: **feel free to resolve this conversation without me.** This is clearer:

```suggestion
    format!("{base_url}{API_BASE_PATH}/{address}")
```

Reply:
You don't have to make this change, I'm letting you know for the future.